packages/catalog-process/src/model/domain/errors.ts

```diff
@@ -249,3 +250,13 @@ export function prettyNameDuplicate(
     title: "Duplicated prettyName",
   });
 }
+
+export function eserviceNotActive(
+  eserviceId: EServiceId
+): ApiError<ErrorCodes> {
+  return new ApiError({
+    detail: `EService ${eserviceId} is not active`,
+    code: "eserviceNotActive",
+    title: "EService is not published",
+  });
+}
```

Comment:
I am not sure about adding the "active" adjective for the eservice, which can be confused with a descriptor state.

Reply:
Proposed solution:
- delete the [function](https://github.com/pagopa/interop-be-monorepo/pull/732/files#diff-3f503cc95ca009a23d5d9f1427d2b7e0b576843a1770756967a3ccd19b803608R105-R113) below and move the check inside the business logic 
- rename the error to `descriptorStateDoesNotAllowDescriptionUpdate` or something similar, or use the already existing error `notValidDescriptorState` (which by the way could be renamed to `incompatibleDescriptorState`, otherwise "not valid" might seem that it's completely out of the states handled by the platform)

Reply:
> Proposed solution:
> 
>     * delete the [function](https://github.com/pagopa/interop-be-monorepo/pull/732/files#diff-3f503cc95ca009a23d5d9f1427d2b7e0b576843a1770756967a3ccd19b803608R105-R113) below and move the check inside the business logic
> 
>     * rename the error to `descriptorStateDoesNotAllowDescriptionUpdate` or something similar, or use the already existing error `notValidDescriptorState` (which by the way could be renamed to `incompatibleDescriptorState`, otherwise "not valid" might seem that it's completely out of the states handled by the platform)

I can't use that error since it also takes a descriptorId that I don't have, so I renamed that one that I created, I also removed the function as you suggested